src/Prism.Core/Common/ParametersExtensions.cs

```diff
@@ -108,6 +139,12 @@ private static bool TryGetValueInternal(KeyValuePair<string, object> kvp, Type t
             return success;
         }
 
+        /// <summary>
+        /// Checks to see if key exists in IEnumberable of KeyValuePairs
```

Comment:
Let's not reference the KeyValuePairs

Reply:
I removed IEnumberable of KeyValuePairs and replaced it with parameter collection